src/OrchardCore.Modules/OrchardCore.Media/Controllers/AdminController.cs

```diff
@@ -272,6 +272,11 @@ public async Task<IActionResult> MoveMedia(string oldPath, string newPath)
                 return NotFound();
             }
 
+            if (!_allowedFileExtensions.Contains(Path.GetExtension(newPath), StringComparer.OrdinalIgnoreCase))
```

Comment:
This is conflict with the work that you done here #4942? Correct me if i'm wrong?

Reply:
No, we don't want to get rid of file extension check, we just want it in admin UI.